src/patternfly/components/Radio/radio.scss

```diff
@@ -17,7 +17,7 @@
   // * Radio input
   --#{$radio}__input--first-child--MarginInlineStart: #{pf-size-prem(1px)};
   --#{$radio}__input--last-child--MarginInlineEnd: #{pf-size-prem(1px)};
-  --#{$radio}__body--MarginBlockStart: var(--pf-t--global--spacer--sm);
+  --#{$radio}__body--MarginBlockStart: var(--pf-t--global--spacer--gap--group--vertical);
```

Comment:
Have you confirmed this with design? Doesn't seem like the right use, especially since there is also a gap on the radio component that combines with this margin

<img width="376" alt="Screenshot 2024-07-15 at 9 05 33 PM" src="https://github.com/user-attachments/assets/2deed89b-a2b1-4644-8e00-616d3ff99a7a">

Reply:
TODO: followup with Lucia

Reply:
I'm gonna say original token for this one. I think the gap token would be used for the space between the radio component and an input label